apps/cms/src/app/app.component.ts

```diff
@@ -1,14 +1,11 @@
 import { Component } from '@angular/core';
 import { RouterModule } from '@angular/router';
-import { NxWelcomeComponent } from './nx-welcome.component';
 
 @Component({
   standalone: true,
```

Comment:
🤔 I see that the migration removes explicit standalone flags. Doesn’t that mean the standalone option for components is true by default now?